cmd/crates/soroban-test/tests/it/integration/tx/operations.rs

```diff
@@ -0,0 +1,414 @@
+        ])
+        .assert()
+        .success();
+    let after = client.get_account(&test).await.unwrap();
+    assert_eq!(before.signers.len() + 1, after.signers.len());
+    // Now remove signer with a weight of 0
+    sandbox
+        .new_assert_cmd("tx")
+        .args([
+            "new",
+            "set-options",
+            "--signer",
+            test1.as_str(),
+            "--signer-weight",
+            "0",
+        ])
+        .assert()
+        .success();
+    let after = client.get_account(&test).await.unwrap();
+    assert_eq!(before.signers.len(), after.signers.len());
+}
+
+#[tokio::test]
+async fn set_options() {
+    let sandbox = &TestEnv::new();
+    let client = soroban_rpc::Client::new(&sandbox.rpc_url).unwrap();
+    let (test, alice) = setup_accounts(sandbox);
+    let before = client.get_account(&test).await.unwrap();
+    assert!(before.inflation_dest.is_none());
+    sandbox
+        .new_assert_cmd("tx")
+        .args([
+            "new",
+            "set-options",
+            "--inflation-dest",
+            test.as_str(),
+            "--home-domain",
+            "test.com",
+            "--master-weight=100",
+            "--med-threshold=100",
+            "--low-threshold=100",
+            "--high-threshold=100",
+            "--signer",
+            alice.as_str(),
+            "--signer-weight=100",
+            "--set-required",
+            "--set-revocable",
+            "--set-clawback-enabled",
+            "--set-immutable",
+        ])
+        .assert()
+        .success();
+    let after = client.get_account(&test).await.unwrap();
+    println!("{before:#?}\n{after:#?}");
+    assert_eq!(
+        after.flags,
+        xdr::AccountFlags::ClawbackEnabledFlag as u32
+            | xdr::AccountFlags::ImmutableFlag as u32
+            | xdr::AccountFlags::RevocableFlag as u32
+            | xdr::AccountFlags::RequiredFlag as u32
+    );
+    assert_eq!([100, 100, 100, 100], after.thresholds.0);
+    assert_eq!(100, after.signers[0].weight);
+    assert_eq!(alice, after.signers[0].key.to_string());
+    let xdr::PublicKey::PublicKeyTypeEd25519(xdr::Uint256(key)) = after.inflation_dest.unwrap().0;
+    assert_eq!(test, stellar_strkey::ed25519::PublicKey(key).to_string());
+    assert_eq!("test.com", after.home_domain.to_string());
+    sandbox
+        .new_assert_cmd("tx")
+        .args([
+            "new",
+            "set-options",
+            "--inflation-dest",
+            test.as_str(),
+            "--home-domain",
+            "test.com",
+            "--master-weight=100",
+            "--med-threshold=100",
+            "--low-threshold=100",
+            "--high-threshold=100",
+            "--signer",
+            alice.as_str(),
+            "--signer-weight=100",
+            "--set-required",
+            "--set-revocable",
+            "--set-clawback-enabled",
+        ])
+        .assert()
+        .failure();
+}
+
+#[tokio::test]
+async fn change_trust() {
```

Comment:
I don't see this test doing anything with the change trust op. What am I missing?

Reply:
It's used in the issue_asset function. But I can add an additional check for a third account.